app/models/miq_server.rb

```diff
@@ -487,11 +487,24 @@ def self.zone_is_modifiable?
     Zone.visible.in_my_region.count > 1
   end
 
-  def self.audit_managed_resources
-    total_vms   = VmOrTemplate.active.count
-    total_hosts = Host.active.count
+  def self.managed_resources
+    {
+      :vms                     => VmOrTemplate.active.count,
+      :hosts                   => Host.active.count,
+      :aggregate_physical_cpus => MiqRegion.my_region.aggregate_physical_cpus(Host.active),
```

Comment:
This one is relatively expensive, considering accepting a parameter to indicate what to collect so e.g. https://github.com/ManageIQ/manageiq/pull/19857/files#diff-c6351ff454ebabe0a2a4a762d1edcfdeR507 can say it just wants `:vms, :hosts`